src/DSA/Map/Map_test.go

```diff
@@ -0,0 +1,38 @@
+package mapp
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMap(t *testing.T) {
+	mapp := NewMap[string, int]()
+
+	mapp.Set("foo", 55)
+	assert.Equal(t, 1, mapp.Len(), "Expected: 1, got: %v", mapp.Len())
+
+	mapp.Set("fool", 75)
+	assert.Equal(t, 2, mapp.Len(), "Expected: 2, got: %v", mapp.Len())
+
+	mapp.Set("foolish", 105)
+	assert.Equal(t, 3, mapp.Len(), "Expected: 3, got: %v", mapp.Len())
+
+	mapp.Set("bar", 69)
+	assert.Equal(t, 4, mapp.Len(), "Expected: 4, got: %v", mapp.Len())
+
+	result := mapp.Get("bar")
+	assert.Equal(t, 69, result, "Expected: 69, got: %v", result)
+
+	result = mapp.Get("blaz")
+	assert.Nil(t, result, "Expected: nil, got: %v", result)
```

Comment:
https://github.com/nacknime-official/kata-machine-go/pull/2/files#r1369979568

Reply:
`mapp.Delete` also returns a value, but it's never checked if deleting an unset key returns `nil`, the current test cases only check if the map is the same length after deleting an unset key. Should I modify `mapp.Delete` to also return a bool and add a test case for it?

Reply:
I'm beginning to think that I had a wrong assumption all along and `Delete` should not return any value. I'll make it return only a bool instead.